llvm/test/CodeGen/BPF/acquire-release.ll

```diff
@@ -0,0 +1,106 @@
+; RUN: llc < %s -march=bpfel -mcpu=v4 -verify-machineinstrs -show-mc-encoding \
+; RUN:   | FileCheck -check-prefixes=CHECK-LE %s
+; RUN: llc < %s -march=bpfeb -mcpu=v4 -verify-machineinstrs -show-mc-encoding \
+; RUN:   | FileCheck -check-prefixes=CHECK-BE %s
+
+; Source:
```

Comment:
This IR file is simple enough, thus doesn't need the source.

Usually when testing a single instruction, we just write the IR files by hands, instead of generating them from clang.